src/builder-git.c

```diff
@@ -148,6 +148,83 @@ git_has_version (int major,
   return TRUE;
 }
 
+static gboolean
+git_lfs (GFile   *dir,
+         char   **output,
+         GSubprocessFlags flags,
+         GError **error,
+         ...)
```

Comment:
I would just take a null terminated `const char **` here and leave the vaargs to builder_git_run_lfs

Reply:
changed in last commit, will squash it later to first one.